tests/test_events.py

```diff
@@ -12,7 +12,7 @@
 from website.addons.base import signals
 from framework.auth import Auth
 from tests import factories
-from tests.base import OsfTestCase
+from tests.base import OsfTestCase, NotificationTestCase
```

Comment:
Is it that we're trying to suppress new behavior that we'd newly get without this change?